codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -1121,19 +1121,19 @@ void addToActiveStreams(DefaultStream stream) {
 
         void removeFromActiveStreams(DefaultStream stream) {
             if (streams.remove(stream)) {
-                try {
-                    // Update the number of active streams initiated by the endpoint.
-                    stream.createdBy().numActiveStreams--;
+                // Update the number of active streams initiated by the endpoint.
+                stream.createdBy().numActiveStreams--;
+            }
+            notifyClosed(stream);
```

Comment:
@nmittler - Good catch.  I'm wondering if this should even be in this method.  Would it make sense in `DefaultStream.close()` after `state = CLOSED`?